### PR TITLE
Add a failing fixtures for RemoveDefaultGetBlockPrefixRector

### PR DESCRIPTION
**Pull request by lyrixx**
ref: https://github.com/rectorphp/rector/issues/7347

### CODE REVIEW

File: tests/Rector/ClassMethod/RemoveDefaultGetBlockPrefixRector/Fixture/extends.php.inc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    {
+        return 'icon';
+    }
+}
+
+class ChoiceType extends AbstractType
+{
+    public function getBlockPrefix()
+    {
+        return 'choice';
```
Comment (lyrixx) [inline tests/Rector/ClassMethod/RemoveDefaultGetBlockPrefixRector/Fixture/extends.php.inc:19]: In SF 3.4, the parent (ChoiceType) **has** this method. [ref](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php#L357-L360)

So If I remove the method in **my** type, I will end up with "choice" and not the default implement in https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/AbstractType.php#L53-L56

File: tests/Rector/ClassMethod/RemoveDefaultGetBlockPrefixRector/Fixture/extends.php.inc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    }
+}
+
+?>
```
Comment (samsonasik) [inline tests/Rector/ClassMethod/RemoveDefaultGetBlockPrefixRector/Fixture/extends.php.inc:47]: you can remove second part if before and after is equal, also, please rename fixture file with `skip_` prefix, eg: `skip_with_get_block_prefix.php.inc`

Comment (lyrixx) [reply tests/Rector/ClassMethod/RemoveDefaultGetBlockPrefixRector/Fixture/extends.php.inc:47]: This fixture is wrong. It's just an example requested by https://github.com/rectorphp/rector/issues/7347

Comment (samsonasik) [reply tests/Rector/ClassMethod/RemoveDefaultGetBlockPrefixRector/Fixture/extends.php.inc:47]: What is the right one?

Comment (lyrixx) [reply tests/Rector/ClassMethod/RemoveDefaultGetBlockPrefixRector/Fixture/extends.php.inc:47]: It's complicated! see this [comment](https://github.com/rectorphp/rector-symfony/pull/212#discussion_r938984638)


But I have an idea!

Comment (lyrixx) [reply tests/Rector/ClassMethod/RemoveDefaultGetBlockPrefixRector/Fixture/extends.php.inc]: Pushed, the test is now failing as expected, and the fixtures is OK, no need to update it.

**Comment by TomasVotruba** [comment]
I've cherry-picked your test fixture here https://github.com/rectorphp/rector-symfony/pull/219

**Pull request closed without merging**